spec/javascripts/packages/document-capture/components/acuant-sdk-spec.js

```diff
@@ -3,10 +3,10 @@
  *
  */
 import { JSDOM } from 'jsdom';
-import AcuantJavascriptWebSdk from '../../../../../public/acuant/11.7.0/AcuantJavascriptWebSdk.min.js';
+import AcuantJavascriptWebSdk from '../../../../../public/acuant/11.7.1/AcuantJavascriptWebSdk.min.js';
 
 const sdkPaths = {
-  '11.7.0': '../../../../../public/acuant/11.7.0/AcuantJavascriptWebSdk.min.js',
+  '11.7.1': '../../../../../public/acuant/11.7.0/AcuantJavascriptWebSdk.min.js',
```

Comment:
Path should be for 11.7.1

```suggestion
  '11.7.1': '../../../../../public/acuant/11.7.1/AcuantJavascriptWebSdk.min.js',
```